### PR TITLE
400 mins free brancher allowance

### PR DESCRIPTION
**Pull request by ikidnapmyself**
Jira: https://teamdotblue.atlassian.net/browse/NL3WEB3-1470

> ### Free Usage Allowance: First 400 Minutes per Hypernode
> 
> You are entitled to a complimentary allocation of 400 minutes for each Hypernode. This means
> you will not be charged for your Brancher minutes usage until it surpasses the threshold of **400 minutes per month
> per Hypernode**, applicable throughout the year 2024. We value your usage and aim to ensure you enjoy the benefits of > our service without incurring additional costs within this specified limit.

### CODE REVIEW

**Review by martijneichelsheim — CHANGES_REQUESTED** [review]

**Comment by iryszawa** [comment]
Placement of the paragraph is good :) 

Small adjustments to the text:

All Combell Openstack and AWS plans (Falcons and Eagles) include 400 complimentary Brancher minutes for each Hypernode as part of the base package. This means you will not be charged for your Brancher minutes usage until it surpasses the threshold of 400 minutes per month per Hypernode. We aim to ensure that you enjoy the benefits of our service without incurring additional costs within this specified limit.

File: docs/hypernode-platform/tools/hypernode-brancher.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
----
+--- 
+
 myst:
-  html_meta:
-    description: 'Hypernode Brancher is a powerful tool that allows users to create
-      and manage temporary servers based on their production Hypernode. Learn more. '
-    title: Hypernode Brancher | Everything you need to know
+html_meta:
```
Comment (tdgroot) [inline docs/hypernode-platform/tools/hypernode-brancher.md:4]: Need to indent these keys properly, see other articles for example

File: docs/hypernode-platform/tools/hypernode-brancher.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+description: 'Hypernode Brancher is a powerful tool that allows users to create
+and manage temporary servers based on their production Hypernode. Learn more. '
+title: Hypernode Brancher | Everything you need to know
 redirect_from:
-  - /en/support/solutions/articles/48001227297-hypernode-brancher/
----
+
+- /en/support/solutions/articles/48001227297-hypernode-brancher/
```
Comment (tdgroot) [inline docs/hypernode-platform/tools/hypernode-brancher.md:10]: This is also badly formatted

**Review by tdgroot — CHANGES_REQUESTED** [review]
Did you format the document in pycharm or something? :sweat_smile:

**Comment by ikidnapmyself** [comment]
@tdgroot i wrote in pycharm then i use mdformat. I also test and build locally with ./bin/ tools

**Review by hrithikkothari1234 — APPROVED** [review]

**Review by tdgroot — APPROVED** [review]

**Pull request merged**